src/h5p-helpers/render.js

```diff
@@ -17,32 +17,30 @@ exports.render = editor => {
             <meta charset="utf-8">
             <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
             <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" crossorigin="anonymous">
+            <script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.8/dist/clipboard.min.js"></script>
```

Comment:
Why do we need a library here? Can't we just use `navigator.clipboard`?

Reply:
The reason why I used library because of browser compatibility handling.
Internet Explorer does not support `navigator.clipboard`. 

We could use `navigator.clipboard` if supporting IE is not required

Reply:
I don't know if we need to support the IE. Maybe @tstaubitz can help here?

We also have the `<input>` with the same information, correct? If IE users can copy this "by-hand" from there, I don't see the need for a library here.

Reply:
@christophblessing please correct me if I'm wrong, but as far as I know we have quit supporting IE with openHPI. In this case, I'd support quitting the support here as well.

Reply:
You are correct, we have. So let's drop the library here.

Reply:
> You are correct, we have. So let's drop the library here.

In that case let me make a new commit removing this library and make a new pull request